.hydra/declarative-jobsets.nix

```diff
@@ -0,0 +1,35 @@
+{ nixpkgs, declInput }:
+
+let
+  pkgs = import nixpkgs {};
+
+  desc = {
+    master = {
+      description = "Build master branch of Redoxpkgs";
+      checkinterval = "60";
+      enabled = "1";
+      nixexprinput = "expr";
+      nixexprpath = "default.nix";
+      schedulingshares = 100;
+      enableemail = false;
+      emailoverride = "";
+      keepnr = 3;
+      hidden = false;
+      type = 0;
+      inputs = {
+        expr = {
+          value = "https://github.com/nix-community/redoxpkgs master";
+          type = "git";
+          emailresponsible = false;
+        };
+      };
+    };
+  };
+
+in {
+  jobsets = pkgs.runCommand "spec-jobsets.json" {} ''
+    cat >$out <<EOF
+    ${builtins.toJSON desc}
+    EOF
+  '';
```

Comment:
why not toFile?

Reply:
yeah, we could use `toFile`. I think I was using `runCommand` because I was also outputing the file to stdout to get the log in Hydra.

Reply:
Fixed by https://github.com/nix-community/redoxpkgs/commit/56c237e02a3ef54fe67885a74bf5db275bbf49ee

P.S. Is there a way I can test Hydra-related stuff locally? Or is it not worth the effort for this repo?

Reply:
@aaronjanse not really. 
But you can build this derivation manually to see what it produces.